### PR TITLE
style(frontend): put buttons at the extreme bottoms for smaller screens [GIX-3015]

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

It was noticed that the buttons in modals for smaller screens are not at the extreme bottom, if the phone is longer.

### Before

Screenshot 2024-09-27 at 10 38 10
Screenshot 2024-09-27 at 10 37 58
Screenshot 2024-09-27 at 10 37 34

### After

Screenshot 2024-09-27 at 10 36 26
Screenshot 2024-09-27 at 10 36 41
Screenshot 2024-09-27 at 10 37 02
Screenshot 2024-09-27 at 10 37 14

### CODE REVIEW

File: src/frontend/src/lib/styles/global/gix.scss [hunk 1]
```diff
@@ -145,7 +145,12 @@ div.modal {
 			border-radius: 0;
 
 			div.content {
+				height: 100vh;
 				padding-inline: var(--padding-2_5x);
+
+				@media (min-width: 768px) {
```
Comment (peterpeterparker) [inline src/frontend/src/lib/styles/global/gix.scss:151]: `@include media.min-width(medium) {`

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/styles/global/gix.scss:151]: thank you!

File: src/frontend/src/lib/styles/global/gix.scss [hunk 2]
```diff
@@ -145,7 +145,12 @@ div.modal {
 			border-radius: 0;
 
 			div.content {
+				height: 100vh;
```
Comment (peterpeterparker) [inline src/frontend/src/lib/styles/global/gix.scss:148]: Sorry did not noticed it before, that's correct?

1. The inner content is not 100vh no?
2. On desktop we don't want 100vh?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**